src/plots/layout_attributes.js

```diff
@@ -175,5 +175,12 @@ module.exports = {
         role: 'info',
         editType: 'legend',
         description: 'Determines whether or not a legend is drawn.'
-    }
+    },
+    colorway: {
+        valType: 'colorlist',
+        dflt: colorAttrs.defaults,
+        role: 'style',
+        editType: 'calc',
```

Comment:
Comment for future development: note here that only `pie` traces require this to be `editType: 'calc'`. For most trace types, `editType: 'plot'` would have sufficed and even `editType: 'style'` might have been good enough for cartesian traces. Oh well, we might have to add-trace-type-specific `editType` flags for layout attributes down the road to improve performance.